packages/reactDom/src/ReactServerDOM.ml

```diff
@@ -234,6 +235,13 @@ module Model = struct
         | Fail exn ->
             (* TODO: Can we check if raise is good heres? *)
             raise exn)
+    | React.Function action ->
+        let chunk_id = use_chunk_id context in
+        let action_id =
+          match action.id with Some id -> id | None -> raise (Invalid_argument "Action ID is required")
```

Comment:
if action id is required, why it's an option?

Reply:
Ops, it was present on the previous branch and I didn't pay enough attention on it. Fixing